frontend/src/scenes/surveys/Surveys.tsx

```diff
@@ -50,24 +50,8 @@ function NewSurveyButton(): JSX.Element {
     const { user } = useValues(userLogic)
 
     const button = (
-        <LemonButton
-            to={urls.surveyTemplates()}
-            type="primary"
-            data-attr="new-survey"
-            sideAction={{
-                dropdown: {
-                    placement: 'bottom-start',
-                    actionable: true,
-                    overlay: (
-                        <LemonButton size="small" to={urls.survey('new')}>
-                            Create blank survey
-                        </LemonButton>
-                    ),
-                },
-                'data-attr': 'saved-insights-new-insight-dropdown',
-            }}
-        >
-            New survey
+        <LemonButton to={urls.surveyTemplates()} type="primary" data-attr="new-survey">
```

Comment:
we are losing one functionality here now.
* the arrow expands the drop down and allows you to create a blank survey right away
* the 'new survey' button leads to the template page
* max leads to the max ai view

if the arrow is gone, i believe its not psosible to create a blank survey right away anymore, is that intentional?

Reply:
when you click on "New survey", you can click again to create "Create blank survey"

I thought removing this option is okay, as both still require two clicks anyway, and this way we highlight only Max on this.

![CleanShot 2025-07-31 at 10 16 00](https://github.com/user-attachments/assets/c1762d73-1345-4488-a47a-75ce281b8818)